packages/jest-cli/src/__tests__/watch.test.js

```diff
@@ -89,6 +89,86 @@ describe('Watch mode flows', () => {
     expect(pipe.write.mock.calls.reverse()[0]).toMatchSnapshot();
   });
 
+  it('resolves relative to the package root', async () => {
+    expect(async () => {
```

Comment:
need to `await` this `expect` (if not, just remove async from the top)?

I think the story around async`toThrow` is pretty confusing, I usually have to tweak the code to be certain the test actually tests anyting.

What about `await expect(watch(...)).resolves.toBeDefined()`? Or just `await watch(...)`, without the `expect` (as the test will fail if there are rejected promises)?